### ISSUE TITLE
Unable to capture any calls

### ISSUE THREAD

**Issue by KvMadan** [glowroot/glowroot#656]
I'm trying to instrument with Glowroot and it doesn't provide any details of the method invocations etc. 
Can anyone suggest how to debug it? 
Does glowroot generate any logs?

**Comment by SpencerWoo** [comment]
Yes, check out https://github.com/glowroot/glowroot/blob/main/central/src/main/resources/logback.xml

**Comment by Never-lab** [comment]
Unable to capture any calls — needs framework, agent log, whether any transaction types appear. Without detail → stale. Try 0.14.7.

@nowheresly — close stale / needs-info.
